packages/contracts/src/resources/site.ts

```diff
@@ -37,6 +37,8 @@ import {
 import type { Json, JsonValue } from "./types";
 import { CreatePubRequestBodyWithNulls, jsonSchema } from "./types";
 
+export const TOTAL_PUBS_COUNT_HEADER = "x-total-pubs";
```

Comment:
nice, thanks for explicitly exporting the header name. very annoying we can't set this in `ts-rest` tho